### ISSUE TITLE
The idea of unhandled errors/exceptions should be explained one place

### ISSUE THREAD

**Issue by brianquinlan** [dart-lang/dartdoc#3381]
We talk about unhandled errors/exceptions in a few places:
https://dart.dev/guides/libraries/futures-error-handling
https://dart.dev/articles/archive/zones

It would be nice to have a single place where we discuss the topic in detail and we can link to that documentation from our `Future` API documentation and from those two articles.

**Comment by srawlins** [comment]
I think this is an issue for https://github.com/dart-lang/site-www/issues but I can't transfer over there. @parlough ?

**Comment by parlough** [comment]
Yeah, this should be on site-www, but I don't have permission to transfer either. 

I instead created a new issue (https://github.com/dart-lang/site-www/issues/4732), so you can close this one.

@brianquinlan We probably won't have time to work on this for a while, but once you have a chance, could you respond to that issue with more details on any specific ideas or points that would like to see covered in the consolidated documentation?
